File: backend/src/main/java/com/nectivo/service/BabyService.java
package com.nectivo.service;

import com.nectivo.dto.BabyDto;

import java.util.List;

public interface BabyService {
    BabyDto createBaby(BabyDto dto);
    BabyDto updateBaby(Long id, BabyDto dto);
    BabyDto getBaby(Long id);
    List<BabyDto> getAllBabies();
    void deleteBaby(Long id);
}

File: frontend/src/services/babyService.js
import api from './api';

export const getBabies = () => api.get('/babies');
export const getBaby = (id) => api.get(`/babies/${id}`);
export const createBaby = (data) => api.post('/babies', data);
export const updateBaby = (id, data) => api.put(`/babies/${id}`, data);
export const deleteBaby = (id) => api.delete(`/babies/${id}`);
